noxfile.py

```diff
@@ -1,7 +1,7 @@
 import nox
 
 
-@nox.session(python=["3.7", "3.8", "3.9", "3.10"])
+@nox.session(python=["3.10", "3.11", "3.12"])
```

Comment:
Seeing as 3.8 and 3.9 aren't EOL yet (though the time will come very soon on 3.8) my default assumption would have been to test with those too, but maybe it's not needed?

Reply:
Alright, added 3.9 back.